tests/datadriven/test_variants.py

```diff
@@ -56,11 +57,77 @@ def getDataModelClass(self):
     def getProtocolClass(self):
         return protocol.GAVariantSet
 
-    def verifyVariantsEqual(self, gaVariants, pyvcfVariants):
+    def _compareTwoFloats(self, a, b):
+        afloat = float(a)
+        bfloat = float(b)
+        return (abs(afloat - bfloat) <= 0.001 * abs(bfloat))
+
+    def _compareTwoListFloats(self, a, b):
```

Comment:
This is a subtle problem - I think we should move this to an issue so we can discuss it in isolation and not hold up this PR. I'll open an issue if we agree.